xml/x509_subjects.xml

```diff
@@ -0,0 +1,953 @@
+    <param pos="0" name="hw.device" value="Firewall"/>
+    <param pos="0" name="hw.product" value="Firewall"/>
+    <param pos="0" name="os.vendor" value="pfSense"/>
+    <param pos="0" name="os.product" value="FreeBSD"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=SophosApplianceCertificate">
+    <description>Sophos Appliance</description>
+    <example>CN=SophosApplianceCertificate_C330AC22W713PAF,OU=OU,O=Green House Data,L=Cheyenne,ST=WY,C=US</example>
+    <example>CN=SophosApplianceCertificate_C22315405928-UU5JKO,OU=OU,O=(null),L=NA,ST=NA,C=PK</example>
+    <example>CN=SophosApplianceCertificate,OU=OU,O=Sophos,L=Abingdon,ST=Oxfordshire,C=GB</example>
+    <param pos="0" name="hw.vendor" value="Sophos"/>
+    <param pos="0" name="hw.device" value="Appliance"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=Hubitat Elevation,OU=Hub,O=Hubitat\\, Inc\.,L=Scottsdale,ST=Arizona,C=US$">
+    <description>Hubitat Device Hub</description>
+    <example>CN=Hubitat Elevation,OU=Hub,O=Hubitat\, Inc.,L=Scottsdale,ST=Arizona,C=US</example>
+    <param pos="0" name="hw.vendor" value="Hubitat"/>
+    <param pos="0" name="hw.device" value="Device Hub"/>
+    <param pos="0" name="hw.product" value="Elevation"/>
+    <param pos="0" name="os.vendor" value="Hubitat"/>
+    <param pos="0" name="os.product" value="Linux"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=MAC([a-fA-F0-9]{12}),OU=([^,]+),O=Mercury Security Products\\, LLC,L=Long Beach,ST=CA,C=US$">
+    <description>Mercurity Security (now HID Global)</description>
+    <example hw.product="M5IC" host.mac="000FE507A1F1">CN=MAC000FE507A1F1,OU=M5IC,O=Mercury Security Products\, LLC,L=Long Beach,ST=CA,C=US</example>
+    <example hw.product="EP-1502" host.mac="000FE508BC71">CN=MAC000FE508BC71,OU=EP-1502,O=Mercury Security Products\, LLC,L=Long Beach,ST=CA,C=US</example>
+    <param pos="0" name="hw.vendor" value="Mercury Security"/>
+    <param pos="0" name="hw.device" value="Access Controller"/>
+    <param pos="1" name="host.mac"/>
+    <param pos="2" name="hw.product"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=securelogin.arubanetworks.com,">
+    <description>Aruba Wireless Controller</description>
+    <example>CN=securelogin.arubanetworks.com,O=Aruba Networks,C=US</example>
+    <example>CN=securelogin.arubanetworks.com,OU=Instant,O=Aruba Networks,L=Sunnyvale,ST=California,C=US</example>
+    <param pos="0" name="hw.vendor" value="Aruba Networks"/>
+    <param pos="0" name="hw.device" value="Wireless Controller"/>
+    <param pos="0" name="hw.product" value="Captive Portal"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=Fireware web CA,OU=Fireware,O=WatchGuard$">
+    <description>WatchGuard Firewall</description>
+    <example>CN=Fireware web CA,OU=Fireware,O=WatchGuard</example>
+    <param pos="0" name="hw.vendor" value="WatchGuard"/>
+    <param pos="0" name="hw.device" value="Firewall"/>
+    <param pos="0" name="os.vendor" value="WatchGuard"/>
```

Comment:
`<param pos="0" name="os.cpe23" value="cpe:/o:watchguard:fireware:-"/>`?  Here and elsewhere in these files there appear to be some CPE issues.  How did you add the CPEs?

Reply:
Mostly guess work where I happened to see a similar cpe somewhere else. Happy to remove them all and let the scripts sort it out, if that is a better approach.

Reply:
You should be able to use the script against just these files and it will remove/correct CPEs.  It isn't perfect but should correct most of the issues related to CPEs being introduced here.  We can also do this when landing, which ever you prefer.

Reply:
If yall could handle this while landing, I would appreciate it, thanks!

Reply:
👍